src/main/scala/scalafix/sbt/ScalafixPlugin.scala

```diff
@@ -529,13 +530,29 @@ object ScalafixPlugin extends AutoPlugin {
                 // custom tool classpaths might contain directories for which we would need to stamp all files, so
                 // just disable caching for now to keep it simple and to be safe
                 throw StampingImpossible
+              case "--triggered" =>
+                // If there is a triggered section in a default config file, --triggered flag should be accounted.
```

Comment:
looking at the implementation, it seems to work also for non-default configurations?
```suggestion
                // If there is a triggered section in the config file, --triggered flag should be accounted.
```